vtr_flow/tasks/power/power_frac_sweep/config/config.txt

```diff
@@ -13,22 +13,22 @@ circuits_dir=benchmarks/verilog
 archs_dir=arch/power
 
 # Add circuits to list to sweep
-circuit_list_add=bgm.v
-circuit_list_add=blob_merge.v
-circuit_list_add=boundtop.v
-circuit_list_add=ch_intrinsics.v
-circuit_list_add=diffeq1.v
+#circuit_list_add=bgm.v
+#circuit_list_add=blob_merge.v
+#circuit_list_add=boundtop.v
+#circuit_list_add=ch_intrinsics.v
+#circuit_list_add=diffeq1.v
```

Comment:
Remove all comments and anything modifications to this task. Again if you need, you would create a new one